sky/cli.py

```diff
@@ -1993,12 +1994,12 @@ def logs(
     job_id = None
     if job_ids:
         job_id = job_ids[0]
-        if not job_id.isdigit():
-            raise click.UsageError(f'Invalid job ID {job_id}. '
-                                   'Job ID must be integers.')
     if status:
         job_statuses = core.job_status(cluster, job_ids)
         job_id = list(job_statuses.keys())[0]
+        if job_id is None:
+            click.secho(f'No job found on cluster {cluster!r}.', fg='red')
+            sys.exit(1)
```

Comment:
Can we add a comment here, like `job_id` is None when no job has been submitted to the cluster